src/pages/Search.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SearchInput from '../shared/components/Search';
+import UsersTable from '../shared/components/Table';
+import Header from '../shared/components/Header/Header';
+
+export default function Search() {
+  return (
+    <>
+      <Header/>
```

Comment:
You are repeating `<Header />` in all of your routes. maybe this mean you should have a wrapper and children components?